include/ocpp/v201/utils.hpp

```diff
@@ -52,6 +52,12 @@ std::string sha256(const std::string& str);
 /// @return A SHA256 hash string
 std::string generate_token_hash(const IdToken& token);
 
+/// @brief Align the clock aligned timestamps to the interval values
+/// @param timestamp the timestamp to align
+/// @param align_interval the clock aligned interval to align to since midnight 00:00
+/// @return DateTime type timestamp
```

Comment:
```suggestion
/// \brief Align the clock aligned timestamps to the interval values
/// \param timestamp the timestamp to align
/// \param align_interval the clock aligned interval to align to since midnight 00:00
/// \return DateTime type timestamp
```